config/secrets.yml

```diff
@@ -3,11 +3,13 @@ base: &BASE
   twilio:
     account_sid: <%= ENV['TWILIO_ACCOUNT_SID'] %>
     auth_token: <%= ENV['TWILIO_AUTH_TOKEN'] %>
-    phone_number: '<%= ENV['TWILIO_PHONE_NUMBER'] %>'
+    phone_number: <%= ENV['TWILIO_PHONE_NUMBER'] %>
     workflow_sid: <%= ENV['TWILIO_WORKFLOW_SID'] %>
     workspace_sid: <%= ENV['TWILIO_WORKSPACE_SID'] %>
     call_app_sid: <%= ENV['TWILIO_CALL_APP_SID'] %>
-    voice_number: '<%= ENV['TWILIO_VOICE_NUMBER'] %>'
+    voice_number: <%= ENV['TWILIO_VOICE_NUMBER'] %>
+    api_key: <%= ENV['TWILIO_API_KEY'] %>
+    twilio_secret: <%= ENV['TWILIO_SECRET'] %>
```

Comment:
Make a note to add these to the .env files

Reply:
Sure